src/Moryx.DependentTestModule/ModuleController/ModuleController.cs

```diff
@@ -50,12 +51,13 @@ protected override void OnStart()
         /// <summary>
         /// Code executed when service is stopped
         /// </summary>
-        protected override void OnStop()
+        protected override Task OnStopAsync()
         {
             Thread.Sleep(2000); // Just for system testing.
```

Comment:
```suggestion
             await Task.Delay(2000); // Just for system testing.
```
As you have changed it above as well